src/w3c/headers.js

```diff
@@ -301,6 +305,26 @@ export function run(conf) {
   conf.anOrA = precededByAn.includes(conf.specStatus) ? "an" : "a";
   conf.isTagFinding =
     conf.specStatus === "finding" || conf.specStatus === "draft-finding";
+
+  if (
+    conf.isRecTrack &&
+    !conf.github &&
+    !(
+      conf.otherLinks &&
+      conf.otherLinks.find(linkGroup =>
+        linkGroup.data.find(
+          l =>
+            l.href &&
+            l.href.toString().match(/^https:\/\/github\.com\/.*\/issues/)
+        )
+      )
+    )
```

Comment:
Can we extract this into a separate function `hasGitHubIssuesLink(conf)` or something similar?